lib/foreman_maintain/executable.rb

```diff
@@ -82,6 +91,16 @@ def set_warn(message)
       execution.status = :warning
       execution.output << message
     end
+
+    def set_skip(message)
+      execution.status = :skipped
+      execution.output << message if message && !message.empty?
```

Comment:
`set_{fail,warn,skip,abort}` all look quite the same. Could we have `set_state(state, message)` to define the behavior, and the rest just calling it?